### PR TITLE
Version fix

### PR DESCRIPTION
**Pull request by JaviCerveraIngram**
The packages uploaded to PyPI were not reporting version number correctly. Our setup.py script was getting the version number as `VERSION = environ.get('TRAVIS_TAG')`. While this worked fine when Travis was generating the package and uploading it to PyPI (since TRAVIS_TAG was set to the tagged version that was being built), it was reporting `None` when installing the package from PyPI in the user's system, so version of the installed package was being reported as 0.0.0.

The way it works it has changed slightly. Now, the VERSION variable gets its value from the contents of the VERSION file, that is always available. When Travis is building the package from a tagged release, TRAVIS_TAG will be set, and this will cause Travis to overwrite the contents of the VERSION file with the new version.

### CODE REVIEW

**Comment by marcserrat** [comment]
really there is no better way? I believe shall exist a better mechanism, relaying on this constant file for build seams quite fragile and prune to error

File: VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+v15.1
```
Comment (vgrebenschikov) [inline VERSION]: IMHO - bad idea to commit that into git 
file should be generated on build (when there is right environment) and then appear in package
in the way that all relevant tools (like PyCharm) will see it

**Comment by JaviCerveraIngram** [comment]
Commited code with the changes.

File: setup.py [hunk 2]
```diff
@@ -25,7 +26,26 @@
...
 
-VERSION = environ.get('TRAVIS_TAG')
+# Get path to version file
+version_path = join(
+    dirname(abspath(__file__)),
+    'VERSION'
+)
+
+# Try to write version file from Travis tag
+travis_tag = environ.get('TRAVIS_TAG')
```
Comment (vgrebenschikov) [inline setup.py]: IMHO - too complex, and not really fair 

There should be no reference to TRAVIS_TAG in package on pypi (connect-sdk-15.0.tar.gz)
Probably, best solution to write down version into setup.py on build on travis
like:
in git - setup.py looks like:
...
setup(
...
version='0.0.0',
...

in 
.travis.yaml 

script:
- sed -i "s/version = '0.0.0'/version = '${TRAVIS_TAG}'/g" setup.py

Then, setup.py will be modified only while travis build and then package will have just right constant in setup.py
Also, you can also modify connect/__init__.py same way (put __version__ = '0.0.0' in git here)

Comment (vgrebenschikov) [reply setup.py]: as above solution you can check V2 in https://stackoverflow.com/questions/48943503/travis-pypi-package-version

also, probably, it worth to use pbr (V1 by link above)

Comment (vgrebenschikov) [reply setup.py]: probably, better to use 15.dev0 to match python version conventions

Comment (JaviCerveraIngram) [reply setup.py]: Uploaded commit removing TRAVIS_TAG from Python code and using sed. Do we need PBR then? This solves the problem already and does not imply adding an additional dependency.

**Pull request merged**